.rubocop.yml

```diff
@@ -109,6 +108,9 @@ Style/Encoding:
   Exclude:
     - test/rubygems/specifications/foo-0.0.1-x86-mswin32.gemspec
 
+Style/IfInsideElse:
```

Comment:
Alternatively, I'm also fine with disabling this rule in `bundler`. I feel this rule sometimes leads to better code, but sometimes it doesn't, so it's probably not worth bothering.

Reply:
I ended up disabling the rule, but I don't have a strong opinion either way.

Reply:
👍 I agree to disable it on bundler. `IfInsideElse` is too strict for me.